src/ApiConnectors/BaseApiConnector.php

```diff
@@ -161,4 +166,70 @@ protected function getFinderService(): FinderService
     {
         return $this->connection->getAuthenticatedClient(Services::FINDER());
     }
-}
+
+    /**
+     * Convert options array to an ArrayOfString which is accepted by Twinfield.
+     *
+     * @param array $options
+     * @param array|null $forcedOptions
```

Comment:
Can you add a comment to explain what the 2nd argument does?

Reply:
In some cases you are not allowed to change certain options (such as the dimtype, which should always be DEB when using `CustomerApiConnector ListAll`), in which case the `$forcedOptions` parameter will be set by the `ApiConnector` for this option, which will override any user settings in `$options`

Added to description